widgets/modules/EnsEMBL/Web/Component/AFDB.pm

```diff
@@ -0,0 +1,78 @@
+=head1 LICENSE
```

Comment:
I dont understand why you need this module (you can put all of this in the module `widgets/modules/EnsEMBL/Web/Component/Transcript/AFDB.pm` ). Maybe something to discuss in the group review

Reply:
so the only reason you would have this module is if you want to have the alphafold widget on a different page/component but it seems its used only one place.

Reply:
I just mirrored what has been done for the pdbe widget on the perl side.